src/components/site/ChartLayout.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import API from './API';
+import InputBox from './InputBox';
+import ScatterPlot from './ScatterPlot';
+
+class Practice extends React.PureComponent {
```

Comment:
Change the name of your class to match the name of your file.  It's working right now in App.jsx to import `Chartlayout` even though there isn't a class of that name because of the way `export default` works.  But for understandability, in react, and javascript in general, it's best practice to match your file names with your class (I mean in cases like this where the whole file is just the class).